src/Symfony/Component/Notifier/Bridge/Iqsms/README.md

```diff
@@ -0,0 +1,23 @@
+Iqsms Notifier
+==============
+
+Provides [Iqsms[(https://iqsms.ru) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```ini
```

Comment:
Sorry, please remove the `ini` and close

Reply:
ok